controller/routes.go

```diff
@@ -46,6 +46,9 @@ func (h *Handler) InitRoutesGin() *gin.Engine {
 		orders.GET("/bymonth", h.GetCourierCompletedOrdersByMonth)
 		orders.GET("/:id", h.GetOrders)
 		orders.PUT("/:id", h.UpdateOrder)
+		orders.GET("/service/completed", h.GetAllCompletedOrdersOfCourierService)
+		orders.GET("/service/completed/bydate", h.GetAllCompletedOrdersOfCourierServiceByDate)
+		orders.GET("/service/completed/bycourierid", h.GetAllCompletedOrdersOfCourierServiceByCourierId)
```

Comment:
I would say that you need to have all of this as single controller that just accepts different parameters